Updated instructions for DurableTask nuget package in VS 2017

desinole opened the pull request:
Alleviates the need for editing the .csproj file in VS2017. Directly searches for the beta version of the nuget package and adds it to the project.

azurecla commented:
Hi __@erikaehrli__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

ggailey777 commented:
@erikaehrli please work with @tdykstra on this addition.

ggailey777 commented:
Closing PR due to no response. Please reopen and contact @tdykstra on how to best make these additions.

desinole committed:
Updated instructions for DurableTask nuget package in VS 2017 

Alleviates the need for editing the .csproj file in VS2017. Directly searches for the beta version of the nuget package and adds it to the project.

cgillum reviewed (approved):
I personally prefer to edit the .csproj directly, but this addition is certainly better than the outdated one we had before, so I'm going to go ahead and approve this. :)

#sign-off
